src/data/store.test.js

```diff
@@ -60,17 +60,17 @@ describe('store aggregator module', () => {
         });
       });
       describe('if no SEGMENT_KEY', () => {
-        const key = configuration.SEGMENT_KEY;
+        const key = getConfig().SEGMENT_KEY;
         beforeEach(() => {
-          configuration.SEGMENT_KEY = false;
+          getConfig().SEGMENT_KEY = false;
         });
         it('exports thunk and logger middleware, composed and applied with dev tools', () => {
           expect(createStore().middleware).toEqual(
             composeWithDevTools(applyMiddleware(thunkMiddleware, createLogger())),
           );
         });
         afterEach(() => {
-          configuration.SEGMENT_KEY = key;
+          getConfig().SEGMENT_KEY = key;
         });
```

Comment:
I'm looking for help with this case. I'm not sure how to change the result of `getConfig()` for just one test case.

Reply:
Currently it fails with:
```
 FAIL  src/data/store.test.js
  ● store aggregator module › exported store › middleware › if no SEGMENT_KEY › exports thunk and logger middleware, composed and applied with dev tools

    expect(received).toEqual(expected) // deep equality

    - Expected  - 0
    + Received  + 4

      Object {
        "withDevTools": Object {
          "applied": Array [
            "thunkMiddleware",
            "logger",
    +       Object {
    +         "map": "EVENTS MAP",
    +         "model": "Segment",
    +       },
          ],
        },
      }

      66 |         });
      67 |         it('exports thunk and logger middleware, composed and applied with dev tools', () => {
    > 68 |           expect(createStore().middleware).toEqual(
         |                                            ^
      69 |             composeWithDevTools(applyMiddleware(thunkMiddleware, createLogger())),
      70 |           );
      71 |         });

      at Object.<anonymous> (src/data/store.test.js:68:44)

```
I presume this is because `getConfig().SEGMENT_KEY = false` isn't actually setting `SEGMENT_KEY` to `false` within the test body.